Framework/Core/include/Framework/DataTypes.h

```diff
@@ -80,7 +80,8 @@ namespace o2::aod::mcparticle::enums
 enum MCParticleFlags : uint8_t {
   ProducedByTransport = 0x1,
   FromBackgroundEvent = 0x2, // Particle from background event (may have been used several times)
-  PhysicalPrimary = 0x4      // Particle is a physical primary according to ALICE definition
+  PhysicalPrimary = 0x4,     // Particle is a physical primary according to ALICE definition
+  FromOutOfBunchPileUpCollision = 0x8
```

Comment:
Could you add a comment, as for the others? Should we write that this is only for Run 2 for now?

Reply:
Done, I also specified this is only for run 2 at the moment